client/src/Movies/SavedList.js

```diff
@@ -12,7 +13,11 @@ export default class SavedList extends Component {
         {this.props.list.map(movie => (
           <span className="saved-movie">{movie.title}</span>
         ))}
-        <div className="home-button">Home</div>
+        <div className="home-button">
+          <Link to="/">
```

Comment:
Great work linking it back to the home page! I now challenge you to look into NavLinks, which come with the ability to style the current/active link